index.html

```diff
@@ -1,35 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Can we keep any kind of demo?

Reply:
I don't see the point since tangram-css is relatively low-level, I think that the demo should be at tangram.cartodb. There is an usage example in the Readme that could be used instead.

Reply:
👍